config/mainCfg_TauTau_UL18_lxplus.cfg

```diff
@@ -34,7 +34,7 @@ regions    = SR, SStight, OSrlx, SSrlx, OSinviso, SSinviso
 
 [configs]
 
-sampleCfg = config/sampleCfg_TauTau_UL18.cfg
+sampleCfg = config/sampleCfg_UL18.cfg
```

Comment:
I think it should be `config/sampleCfg_UL18_lxplus.cfg`

Reply:
The one above works at LLR. It the new ```*_lxplus``` one, using ```/eos/```, supposed to be the new default?

Reply:
this config should be exclusively for `lxplus`. I suppose we could/should in fact remove that file entirely to avoid unnecessary duplications, and use `config/mainCfg_TauTau_UL18.cfg` both at LLR and on lxplus though. @johanWulff any reason I would be missing to use this cfg file specifically?

Reply:
No there's no reason to have 2 main_cfgs. The only thing that is in fact lxplus specific is the paths to the datasets so that should only affect sampleCfg.